src/librustc_session/options.rs

```diff
@@ -947,6 +947,10 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         (default: PLT is disabled if full relro is enabled)"),
     polonius: bool = (false, parse_bool, [UNTRACKED],
         "enable polonius-based borrow-checker (default: no)"),
+    polymorphize: bool = (true, parse_bool, [TRACKED],
+          "perform polymorphization analysis"),
+    polymorphize_errors: bool = (false, parse_bool, [TRACKED],
+          "emit errors from polymorphization analysis for debugging"),
```

Comment:
Could this be some `#[rustc_...]` attribute instead? I'm guessing no?

Reply:
I've removed `-Z polymorphize-errors` and added a `#[rustc_polymorphize_error]` attribute which is applied to functions and will result in errors being emitted for those functions and their closures.